managed/services/grafana/client.go

```diff
@@ -183,26 +183,26 @@ func (r role) String() string {
 
 // getUserID returns user ID from Grafana for given user
 func (c *Client) GetUserID(ctx context.Context) (int, error) {
-	userID := -1
+	userID := 0.0
 
 	authHeaders, err := c.authHeadersFromContext(ctx)
 	if err != nil {
-		return userID, err
+		return 0, err
 	}
 
 	var m map[string]interface{}
 	err = c.do(ctx, "GET", "/api/user", "", authHeaders, nil, &m)
 
 	if err != nil {
-		return userID, err
+		return 0, err
 	}
 
-	val, ok := m["id"].(float64)
+	userID, ok := m["id"].(float64)
```

Comment:
interesting case, why do we cast to float instead of int?

Reply:
We don't dictate how it gets **unmarshaled**. I noticed it gets read as **float64** instead of an **int**.

Reply:
we can dictate by using structs instead of map :) But ok